Code.gs

```diff
@@ -96,6 +96,7 @@ function main(){
   if(targetCalendar == null){
       Logger.log("Creating Calendar: " + targetCalendarName);
       targetCalendar = CalendarApp.createCalendar(targetCalendarName);
+      targetCalendar.setSelected(true); //Sets the calendar as "shown" in the Google Calendar UI
```

Comment:
Isn't that imposing on the user?

Reply:
Possibly, but I have a feeling we would get more complaints of "it's not creating any events" [lots of troubleshooting later...] "oh, the calendar just wasn't shown in Google Calendar"